### ISSUE TITLE
bug in instanceof

### ISSUE THREAD

**Issue by janzimmermannnetrex** [krakjoe/uopz#98]
We observe that the php operator `instanceof` is not working properly with uopz 5.1.0. It fails with fatal error once the class doesn't exist. It should just return false based on documentation. It is working as expected once we downgrade to uopz 5.0.2. See example code bellow.

```php
class MyClass {
}
$obj = new MyClass();
$classname = "ClassWhichDoesntExist";
var_export($obj instanceof $classname);  // fatal error ...
```
PHP info: [phpi-oss.txt](https://github.com/krakjoe/uopz/files/2733289/phpi-oss.txt)

**Comment by krakjoe** [comment]
Please try that latest sources in master branch.

**Comment by krakjoe** [comment]
Fixed in 6.0.0
